src/models/service.ts

```diff
@@ -93,9 +67,7 @@ const ServiceMetadataO = t.partial({
 
   email: t.string,
 
-  pec: t.string,
-  // The attribute `scope` should be required, but it's currently optional for backward-compatibility reasons
-  scope: enumType<ScopeEnum>(ScopeEnum, "scope")
+  pec: t.string
```

Comment:
may you change all `t.string` into `NonEmptyString` ? (since the field is not required we can omit it in case it's e)